spec/reek/smells/nested_iterators_spec.rb

```diff
@@ -43,7 +75,7 @@ def chained
         @sig.keys.sort_by { |xray| xray.to_s }.each { |min| md5 << min.to_s }
       end
     EOS
-    expect(src).not_to reek_of(:NestedIterators)
+    expect(src).not_to reek_of(described_class)
```

Comment:
I have a slight preference for using the symbol, because it means you need a little less context to understand the spec.

Also, I prefer the idea of code reeking of a smell over it reeking of a detector class. OTOH, I wouldn't mind specs written like:

```
let(:detector) { described_class.new }

it 'something' do
  expect(detector).to detect_smell_in(source)
end
```

Just my 2 cents of the evening.